user-scripts/pathoschild.stewardscript.js

```diff
@@ -145,6 +145,12 @@
                                 .text("crossactivity")
                                 .attr({ href: "//tools.wmflabs.org/meta/crossactivity/" + encodeURIComponent(user), title: "Pathoschild's CrossActivity (measures a user's latest edit, bureaucrat, or sysop activity on all wikis)" })
                             )
+                            .append(", ")
+                            .append(this
+                                .Make("a")
+                                .text("global user rights")
+                                .attr({ href: "//meta.wikimedia.org/wiki/Special:GlobalUserRights" + encodeURIComponent(user), title: "Global User Rights" })
```

Comment:
There's a missing slash between `Special:GlobalUserRights` and the username here.

Reply:
Oh, that's right. I'll amend the patch. Thanks for reviewing :)

Reply:
Fixed on 01cbf01 I think.